src/service/config.py

```diff
@@ -0,0 +1,52 @@
+"""
+Configuration settings for the Spark Manager API.
+"""
+
+import logging
+import os
+from functools import lru_cache
+
+from pydantic import Field
+from pydantic_settings import BaseSettings
+
+APP_VERSION = "0.0.1"
+
+
+class Settings(BaseSettings):
+    """
+    Application settings.
+
+    Values will be loaded from environment variables or use the provided defaults.
+    """
+
+    app_name: str = "CDM Spark Cluster Manager API"
+    app_description: str = "API for managing Spark clusters in Kubernetes"
+    api_version: str = APP_VERSION
+    log_level: str = Field(
+        default=os.getenv("LOG_LEVEL", "INFO"), description="Logging level"
+    )
```

Comment:
If I'm reading the pydantic docs correctly, these fields are set by the corresponding env var - so app_name would be set by the `app_name` environment variable, log_level by the `log_level` env var, etc. Is that what you want?

Reply:
only the logging level is set by env variable - I was thinking we could use different logging level for different environment. Other than that, the other informational strings are fixed.

Reply:
Why are you using the BaseSettings class then? The whole point of that, IIUC, is that it populates its keys from the enviornment

Reply:
And I think it will override the default values:

> If you create a model that inherits from BaseSettings, the model initialiser will attempt to determine the values of any fields not passed as keyword arguments by reading from the environment. (Default values will still be used if the matching environment variable is not set.)

from https://docs.pydantic.dev/latest/concepts/pydantic_settings/

Reply:
okay. yea. Removed pydantic_settings - maybe I was thinking setting it with the envionment variables than changed my mind. But doesn't really matter here.